tests/README.md

```diff
@@ -1,29 +1,34 @@
-# TrustyAI OpenShift CI
+# TrustyAI OpenShift CI and E2E Tests
 The TrustyAI OpenShift-CI tests are based off the [odh-manifests CI tests](https://github.com/opendatahub-io/odh-manifests/tree/master/tests).
 
-# Running containerized tests
+# Running locally
 
-Running the tests this way assumes that you have an active kubeadmin login
-on the cluster that you want to run the tests against and that you have podman
-installed.  (If you prefer docker, you can edit the Makefile to replace podman
-with docker).
+Assuming you are oc login'd with admin rights to a cluster, run the following from within the `tests` directory:
 
-Run the following:
+```sh
+make test
+```
 
+This will build the test container, run those tests against the cluster you are logged into, and then clean up afterwards.
+This is equivalent to running:
 ```sh
-cd tests
-make build
-make run
+make build # build the test container (necessarily after any changes to test scripts/resources
+make run   # run the tests against the existing cluster
+make clean # remove the artifacts of the test from the cluster (operator, ODH, projects, etc)
 ```
 
-## Cleaning up after your test run (optional)
 
-Only run the following if you want to eliminate your Open Data Hub installation.
 
-To cleanup the Open Data Hub installation after a test run, you can run `make clean`.
-Running `make clean` **will wipe your Open Data Hub installation** and delete the project.
+## Useful Arguments
+* `BUILD_TOOL=docker/podman`: set the tool used to build and run the testing container
+* `SKIP_INSTALL=true/false`: skip the install of the ODH operator, if you've already installed it manually or via a previous test
+* `SKIP_KFDEF_INSTALL=true/false`: skip the install of ODH via KFdef, if you've already installed it manually or via a previous test
```

Comment:
Side note more for a future ticket, we should verify if this works with RHODS operator too. I guess so but it should be better to check it